File: data/petstore/oseg_generated/csharp/UpdatePetWithForm_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class UpdatePetWithFormDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var apiCaller = new PetApi(config);

            apiCaller.UpdatePetWithForm(
                petId: null,
                name: null,
                status: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Pet#UpdatePetWithForm: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: data/petstore/oseg_generated/php/UpdatePetWithForm_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    $api_caller = new OpenAPI\Client\Api\PetApi(config: $config);

    $api_caller->updatePetWithForm(
        pet_id: null,
        name: null,
        status: null,
    );
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling Pet#updatePetWithForm: '
        . print_r($e->getResponseObject());
}
